app/controllers/api/v1/about_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module V1
+    class AboutController < ApiController
+      def index
+        render json: About.first
```

Comment:
If the admin adds a new text, shouldn't be better to get the last record not the first one?

```suggestion
        render json: About.last
```